playground/nuxt.config.ts

```diff
@@ -53,6 +53,7 @@ export default defineNuxtConfig({
       baseURL: 'https://netlify-photo-gallery.netlify.app'
     },
     layer0: {},
```

Comment:
Shall we remove older layer0?

Reply:
I think there was no baseURL specified before, shall I update both the configs to have one?

Reply:
Good idea